### PR TITLE
Update 'html-react-parser' dependency

### PR DESCRIPTION
**Pull request by alexkb0009**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+legacy-peer-deps=false
```
Comment (alexkb0009) [inline .npmrc:1]: Relevant for NPM v7+ & currently unused (we're on NPM <=v6)

**Comment by alexkb0009** [comment]
Ready for review I think. @utku-ozturk Might be worth checking out https://github.com/dbmi-bgm/cgap-portal/pull/475/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519 -- and perhaps replicating to some extent (at minimum we should define auth0-lock in CGAP/4DN now, instead of in SPC -- Auth0-lock wanted React15 and was giving some warnings/errors on NPM7 re: the conflicts (us wanting React16+)).

**Comment by alexkb0009** [comment]
(OffTopic) Later, in context of Node14+/NPM7+ upgrade, will checkout this ['resolve.alias' approach](https://blog.maximeheckel.com/posts/duplicate-dependencies-npm-link/).

**Review by Bianca-Morris — APPROVED** [review]
Changes look great. I did test only on CGAP though (with the `html-parser-update` branch), so the Table of Contents stuff may need another set of eyes.

**Review by utku-ozturk — APPROVED** [review]
It looks and works great! To test it on 4DN, we released a draft package (https://github.com/4dn-dcic/shared-portal-components/releases/tag/0.1.21b4) and deployed it into hotseat. All manual and Cypress tests succeeded.

**Pull request merged**